app/pods/components/content-search/component.js

```diff
@@ -0,0 +1,62 @@
+import Component from "@ember/component";
+import { later } from "@ember/runloop";
+import { task, timeout } from "ember-concurrency";
+import { action } from "ember-decorators/object";
+import { alias } from "ember-decorators/object/computed";
+import { service } from "ember-decorators/service";
+
+export default class SearchBoxComponent extends Component {
+  tagName = 'span'
+  hideResultsBox = true
+  qs = ''
+  @alias('searchTask.lastSuccessful.value') results
+
+  @service router
+  @service store
+
+  searchTask = task(function*() {
+    yield timeout(1000);
+    const searchQuery = this.get("qs");
+    const contents = this.get("store").peekAll("content");
```

Comment:
no this will list ALL of the contents in store, we only want those related to `this` runAttempt